config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml

```diff
@@ -1,9 +1,9 @@
-presubmits:  
+presubmits:
   bazelbuild/rules_k8s:
   - name: pull-rules-k8s-e2e
     agent: kubernetes
     cluster: gke_rules-k8s_us-central1-f_testing
-    max_concurrency: 1  # TODO(fejta): increase this
+    max_concurrency: 2
```

Comment:
This restriction is because the tests cannot run concurrently. Did you fix this on the rules_k8s end?

Reply:
Then I misunderstood the meaning of this parameter. Will close the PR for now. Thanks!